src/Starter/database/migrations/2015_11_30_215040_create_role_user_table.php

```diff
@@ -17,6 +17,9 @@ public function up()
             $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
             $table->integer('role_id')->unsigned();
             $table->foreign('role_id')->references('id')->on('roles')->onDelete('cascade');
+            
+            $table->primary(['role_id', 'user_id']);
+
```

Comment:
I'm going to say no given what Jeffrey Way has in his generator tools: https://github.com/laracasts/Laravel-5-Generators-Extended#pivot-tables